src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -573,72 +573,86 @@ query_dfs_mount(const char *path)
 	return idx;
 }
 
-static void
-discover_daos_mount(void)
+/* Discover fuse mount points from /proc/self/mounts and env DAOS_MOUNT_POINT.
+ * Return 0 for success. Otherwise return Linux errno. This function interception
+ * is still effective as long as num_dfs is not zero even if discover_daos_mount_with_env()
+ * returns a non-zero value.
```

Comment:
The code calling discover_daos_mount_with_env() aborts if this returns non-zero which contradicts this comment.

Reply:
The comments are fixed. Thank you!